app/controllers/application_controller.rb

```diff
@@ -38,6 +38,8 @@ def customize_params
     params[:signed_in] = user_signed_in?.to_s
   end
 
+  # This method is used by Devise to decide which is the path to redirect
+  # the user to after a successful log in
```

Comment:
I feel like this comment isn't really needed, I think the method name is pretty self-explanatory.

Reply:
I added because I didn't know it was used TBH. I stumbled upon it casually by doing a debug session step by step. If you grep for it there's no code that calls it, it's called internally by Devise, that's why I added the comment :D